business_requirement/models/business.py

```diff
@@ -199,7 +199,7 @@ def write(self, vals):
         if vals.get('state'):
             br_xml_id = self.env.ref(
                 'business_requirement.group_business_requirement_manager')
-            user = self.env['res.users']
+            user = self.env.user.id
```

Comment:
If you are putting an ID, call the variable `user_id`, but better to always store the recordset in variables and only populate ID when needed through `user.id`